Kinvey/Kinvey/PullOperation.swift

```diff
@@ -19,6 +19,7 @@ internal class PullOperation<T: Persistable>: FindOperation<T> where T: NSObject
         validationStrategy: ValidationStrategy?,
         cache: AnyCache<T>?,
         options: Options?,
+        mustSetRequestResult: Bool = true,
```

Comment:
Under what circumstances is `mustSetRequestResult` set to false?

Reply:
for auto pagination requests, if you set the result for each request you end up holding all instances in memory, that's why the memory consumption was going to the roof, so in this case we should `false` for this. This is all internal code, so no need to the user to set this.